wavefront/config/config.proto

```diff
@@ -22,9 +22,9 @@ syntax = "proto3";
 // $contact_email: veniln@vmware.com
 // $support_link:
 // $source_link: https://github.com/vmware/wavefront-adapter-for-istio
-// $latest_release_link: https://github.com/vmware/wavefront-adapter-for-istio/releases/tag/0.1.1
+// $latest_release_link: https://github.com/vmware/wavefront-adapter-for-istio/releases/tag/0.1.3
 // $helm_chart_link:
-// $istio_versions: "1.0.3, 1.0.4"
+// $istio_versions: "1.4, 1.5, 1.6"
```

Comment:
I think it'd be good to mention the patch versions too.

Reply:
We tested with the latest available patch while testing so modified to have .x for the patch.

Reply:
It'd be better to specify the exact patch number instead of x.